app/models/tslr_claim.rb

```diff
@@ -77,6 +83,10 @@ def ineligible_claim_school?
     claim_school.present? && !claim_school.eligible_for_tslr?
   end
 
+  def taught_not_enough_eligible_subjects?
```

Comment:
How about `not_taught_enough_eligible_subjects?` for the method name? "tought not" feels like the words are in the wrong order.r